commcare_connect/templates/opportunity/opportunity_detail.html

```diff
@@ -61,13 +70,15 @@ <h1 class="mb-0">{{ object.name }}</h1>
               </a>
             </li>
           {% endif %}
+          {% if not object.managed or request.org != object.organization %}
```

Comment:
what does the second part do? why is it look for orgs that are not the opportunity's org?

Reply:
I fixed this and made the permissions a little more verbose. [77209bd](https://github.com/dimagi/commcare-connect/pull/380/commits/77209bd0988ca634818e011265ef83e949ce558d)